### PR TITLE
Use solc v0.6.6

### PR DESCRIPTION
**Pull request by moodysalem**

### CODE REVIEW

File: contracts/interfaces/IUniswapV2ERC20.sol [hunk 1]
```diff
@@ -1,20 +1,8 @@
-pragma solidity >=0.5.0;
+pragma solidity >=0.6.0;
```
Comment (NoahZinsmeister) [inline contracts/interfaces/IUniswapV2ERC20.sol:1]: has to be `>=0.6.2` since we're extending

File: contracts/interfaces/IUniswapV2Pair.sol [hunk 2]
```diff
@@ -1,26 +1,8 @@
-pragma solidity >=0.5.0;
+pragma solidity >=0.6.0;
```
Comment (NoahZinsmeister) [inline contracts/interfaces/IUniswapV2Pair.sol:1]: `>=0.6.2`

**Comment by axic** [comment]
@moodysalem @NoahZinsmeister could someone restore the `pr-improvements` or `solc-v0.6.6` branches briefly (Github allows that) so I could copy them off? :)

**Comment by NoahZinsmeister** [comment]
@axic restored! lmk when you've grabbed it

**Comment by axic** [comment]
Thanks, got it!

**Pull request merged**